core/lib/globals_shared.h

```diff
@@ -2017,4 +2017,30 @@ typedef struct _priv_mcontext_t {
 #include "mcxtx.h"
 } priv_mcontext_t;
 
+/* DR_API EXPORT BEGIN */
```

Comment:
Can we limit this to narrower scope than globals_shared.h: I see it can't be limited to instrument.c, but how about os_shared.h?

Reply:
If I remember correctly, I tried to do this, but instrument_api.h does not include os_shared.h and was reluctant to add it just for dr_app_arg_t.

Reply:
instrument_api.h does transitively include os_shared.h: os_exports.h includes os_shared.h and is included by globals.h (limited to core; globals_shared is used by non-core libs) so it should be visible.  Or do you mean it doesn't get published to the API headers by genapi.pl?  I see existing doxygen things in there so it should be.

Reply:
Done.